tgui/packages/tgui/constants.js

```diff
@@ -0,0 +1,226 @@
+/**
+ * @file
+ * @copyright 2020 Aleksej Komarov
+ * @license MIT
+ */
+
+// UI states, which are mirrored from the BYOND code.
+export const UI_INTERACTIVE = 2;
+export const UI_UPDATE = 1;
+export const UI_DISABLED = 0;
+export const UI_CLOSE = -1;
+
+// All game related colors are stored here
+export const COLORS = {
+  // Department colors
+  department: {
+    captain: '#c06616',
+    security: '#e74c3c',
+    medbay: '#3498db',
+    science: '#9b59b6',
+    engineering: '#f1c40f',
+    cargo: '#f39c12',
+    centcom: '#00c100',
+    other: '#c38312',
+  },
+  // Damage type colors
+  damageType: {
+    oxy: '#3498db',
+    toxin: '#2ecc71',
+    burn: '#e67e22',
+    brute: '#e74c3c',
+  },
+};
+
+// Colors defined in CSS
+export const CSS_COLORS = [
+  'black',
+  'white',
+  'red',
+  'orange',
+  'yellow',
+  'olive',
+  'green',
+  'teal',
+  'blue',
+  'violet',
+  'purple',
+  'pink',
+  'brown',
+  'grey',
+  'good',
+  'average',
+  'bad',
+  'label',
+];
+
+export const RADIO_CHANNELS = [
```

Comment:
Not sure this is needed for us. Could probably strip for the first merge and add our own implementation if we need it at a later date?